qa/packages/packages.yaml

```diff
@@ -11,7 +11,6 @@ ceph:
   - python-ceph
   - libcephfs2
   - libcephfs-dev
-  - libcephfs-java
   - libcephfs-jni
```

Comment:
Presumably this should go away too then.

Reply:
done